server/controllers/ReviewsController.js

```diff
@@ -0,0 +1,15 @@
+const ReviewsModel = require('../models/ReviewsModel.js');
+
+const getReviews = (req, res) => {
+  ReviewsModel.getReviews(req, (err, success) => {
```

Comment:
I would only pass the necessary information to the model.getReviews - the main downside given my limited understanding of hardware constraints is that it'll require a lot more memory as the number of requests gets higher. Obviously out of the scope of this project but it's always nice to keep efficiency in mind

Reply:
Agree. Updated.